src/doc/rustdoc/src/unstable-features.md

```diff
@@ -84,6 +84,39 @@ in documentation.
 `#![feature(doc_cfg)]` feature gate. For more information, see [its chapter in the Unstable
 Book][unstable-doc-cfg] and [its tracking issue][issue-doc-cfg].
 
+### `doc_auto_cfg`: Automatically generate `#[doc(cfg)]`
+
+An extension to the `#[doc(cfg)]` feature called `doc_auto_cfg` was added. With it, you don't need
+to add `#[doc(cfg(..)]` anymore unless you want to override the default behaviour. So if we take the
```

Comment:
Changes:

- don't say it "was added"; just say what it is (more consistent with other docs)
- replace `..` with `...` since `..` looks like range syntax

```suggestion
`doc_auto_cfg` is an extension to the `#[doc(cfg)]` feature. With it, you don't need
to add `#[doc(cfg(...)]` anymore unless you want to override the default behaviour. So if we take the
```

Reply:
Done!